### ISSUE TITLE
Web login doesn't make it clear that you need to swipe your MCard to create an account

### ISSUE THREAD

**Issue by westbl** [um-cseg/chez-betty#327]
Users are attempting to get a password (i.e., open an account) using the online portal https://chezbetty.store/login/submit because of ambiguous wording on the page.  The text "This isn't kerberos, so if you don't have a Betty password, you need to use the other form." doesn't indicate that you can't make an account here, and "Or never had one? Use this to get a password for your account." on the "other form" makes it sound like you can create an account using the Forgot Password field.

tl;dr - add text (large text) at the top of the page stating that you must first have an account with Betty (and explain how to make one) before you can use this webpage.
